src/main/java/ch/uzh/ifi/hase/soprafs22/lobby/Lobby.java

```diff
@@ -65,10 +72,13 @@ public void setVisibility(Visibility visibility) {
     }
 
     /**
+     *
      * Create and add a new player
      * @return The created player
+     * @deprecated Use generatePlayer!
      */
     @Override
+    @Deprecated
     public Player addPlayer() {
```

Comment:
We can safely delete a deprecated method since we haven't released our API yet, can't we?

Reply:
yes, I just didn't know if anyone still uses it and didn't want to refactor anything ^^ But it's probably better if I quickly do it